test/e2e/upgrade_test.go

```diff
@@ -30,6 +30,7 @@ const (
 
 func runSimpleUpgradeFlow(test *framework.ClusterE2ETest, updateVersion v1alpha1.KubernetesVersion, clusterOpts ...framework.ClusterE2ETestOpt) {
 	test.GenerateClusterConfig()
+        test.GenerateHardwareConfig()
```

Comment:
I think we would want this in a separate function to avoid running generate hardware for all upgrade tests.
Like here https://github.com/aws/eks-anywhere/blob/main/test/e2e/simpleflow_test.go#L30

Reply:
great catch as usual